app/views/branches/_form_conditionals.html.erb

```diff
@@ -1,17 +1,16 @@
 <%= f.fields_for :conditionals, child_index: conditional_index do |conditional| %>
   <div class="destination">
-    <div class="govuk-form-group <%= conditional.object.errors[:next].empty? ? '' : 'govuk-form-group--error'  %>" data-conditional-index="<%= conditional.index %>">
+    <div class="govuk-form-group <%= conditional.object.errors[:next].empty? ? '' : 'govuk-form-group--error'  %>" data-index-conditional="<%= conditional.index %>">
```

Comment:
i think we deliberately changed this to `data-conditional-index`

Reply:
This would be the conflict I encountered and clearly got wrong (as seen by an earlier change I've already committed as well). Have now corrected.